rails/lib/mock_data/create_default_data.rb

```diff
@@ -954,11 +954,13 @@ def self.create_default_materials_collections
     puts
     puts
     MaterialsCollection.destroy_all
+    Admin::Project.destroy_all
+    default_project = FactoryBot.create(:project, name: 'default data project')
     DEFAULT_DATA[:materials_collections].each do |key, mc|
-      if (mc[:items_count] > 0)
-        FactoryBot.create(:materials_collection_with_items, items_count: mc[:items_count], name: mc[:name], description: mc[:description])
+      if (mc[:items_count])
+        FactoryBot.create(:materials_collection_with_items, mc.merge(project: default_project))
       else
-        FactoryBot.create(:materials_collection, name: mc[:name], description: mc[:description])
+        FactoryBot.create(:materials_collection, mc.merge(project: default_project))
```

Comment:
I tried to simplify these since they were basically just taking the `mc` hash and creating new hash to pass to FactotryBot.create. 
I think this whole file can be refactor to make this whole process more simple. It might be useful for @kevinsantos-cc so the cypress tests are based off this same default data instead of having to recreate lots of the same objects.